### PR TITLE
fix bug in fftw3x_cdft/makefile affecting -fPIC build

### PR DESCRIPTION
**Pull request by boegel**
+ fix inconsistencies w.r.t. `license_file` and `postinstallcmds` while we're at it

cfr. https://github.com/hpcugent/easybuild-easyblocks/issues/977

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyconfigs/i/imkl/imkl-11.3.1.150-iimpi-2016.01-GCC-4.9.3-2.25.eb [hunk 1]
```diff
@@ -16,15 +16,13 @@ checksums = ['b57ff502b5f97f2f783e4bbda7ce42b3']
 
 dontcreateinstalldir = 'True'
 
-# license file
-import os
-license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
+license_file = HOME + '/licenses/intel/license.lic'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/i/imkl/imkl-11.3.1.150-iimpi-2016.01-GCC-4.9.3-2.25.eb]: OS-proof?

Comment (boegel) [inline easybuild/easyconfigs/i/imkl/imkl-11.3.1.150-iimpi-2016.01-GCC-4.9.3-2.25.eb]: What do you mean exactly?

`HOME` is a constant provided by the EasyBuild framework.

If you're referring to using `/` and that not being compatible with Windows, we're a looooong way away from supporting Windows anyway. And probably there the new bash-in-Windows thing is our best option anyway...

**Comment by akesandgren** [comment]
@boegel, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**